core/cloudflow-streamlets/src/main/scala/cloudflow/streamlets/StreamletContext.scala

```diff
@@ -59,7 +62,21 @@ trait StreamletContext {
    * The default bootstrapServers for the Kafka broker that has been installed or configured
    * to be used globally for all Cloudflow applications
    */
-  def internalKafkaBootstrapServers = config.getString("cloudflow.kafka.bootstrap-servers")
+  def internalKafkaBootstrapServers =
+    try {
+      config.getString("cloudflow.kafka.bootstrap-servers")
+    } catch {
+      case e: com.typesafe.config.ConfigException.Missing =>
```

Comment:
Testing for the config path would show the intention cleaner then try/catch, IMHO.